spec/features/bucket_lists/user_edits_bucket_lists_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-feature 'authenticated user creates a bucket list', %(
+feature 'authenticated user edits a bucket list', %(
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.